src/util/transition/ngbTransition.ts

```diff
@@ -26,15 +26,15 @@ export const ngbRunTransition =
     <T>(element: HTMLElement, startFn: NgbTransitionStartFn<T>, options: NgbTransitionOptions<T>):
         Observable<undefined> => {
 
+          // Getting initial context from options
```

Comment:
This is also fixed in my navs PR, but here you're missing tests... let's rollback this one.
My implementation is here → https://github.com/ng-bootstrap/ng-bootstrap/pull/3776/commits/c6b296b7d5e8e4118ff15a17212048775dba5ae6

Reply:
Yes sure. I'll intend to rebase the branch once this part will be delivered. It's just here because I need it to have this PR code working .

It's also required for the carousel. Maybe a separate PR would have been required for this part, to be available for all :)

Reply:
I'll extract my commit into a a separate PR for this, no pb.